requirements.txt

```diff
@@ -1,3 +1,4 @@
 rasa==1.10.2
 protobuf==3.19.4
+importlib-metadata==4.13.0
```

Comment:
Why is this added?

Reply:
Because importlib-metadata releases v5.0.0  which it remove deprecated endpoint. We get error AttributeError: **'EntryPoints' object has no attribute 'get'** and this version doesn't support python 3.7. it's either we upgrade python or downgrade this library. @bearnard